src/markdown-pages/docs/cli-command-reference/cli-command-reference.md

```diff
@@ -12,16 +12,17 @@ To use the Galasa CLI commands you must reference the Galasa bootstrap file or U
 You can [access the code](https://github.com/galasa-dev/cli) and [download the CLI tool](https://github.com/galasa-dev/cli/releases) from the _cli_ repository in GitHub. The following versions of the Galasa CLI tool are available to download for different operating systems:
 
 | Operating system  |  Download  |
-| :---- | :-------- |
+| :---- | :-------- | 
 | MacOSX  | galasactl-darwin-amd64 |
+| MacOSX  | galasactl-darwin-arm64 |
 | Linux amd64 | galasactl-linux-amd64 | 
 | zLinux  | galasactl-linux-s390x | 
 | Windows | galasactl-windows-amd64.exe | 
 
 
 ## The CLI commands
 
-The following section provides a summary of current commands, along with a brief description. Click the links for more detailed information and examples on how to use the commands.
+The following section provides a summary of current commands, along with a brief description. Click the links for more detailed information and examples on how to use these commands.
```

Comment:
Might be worth pointing where a full reference of the syntax can be found from this point ?

Reply:
have added link to cli repo